src/index.html

```diff
@@ -17,6 +30,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index/html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
Ensure that the path `images/logo.png` is correct and that the image exists at this location.